src/core/BufferAttribute.js

```diff
@@ -32,6 +32,12 @@ function BufferAttribute( array, itemSize, normalized ) {
 
 }
 
+Object.defineProperty( BufferAttribute.prototype, "needsUpdate", {
+
+	set: function(value) { if ( value === true ) this.version ++; }
+
+});
```

Comment:
It is a setter for one lined function, but yes i can reformat this correctly. And in fact it seems that all my change were erase by merging... except the conflict files -_-' ! Damned

I close this one the time to got a stable version (with good formating ;)